ui/components/ui/metafox-logo/metafox-logo.component.js

```diff
@@ -33,19 +36,23 @@ export default class MetaFoxLogo extends PureComponent {
               ? getBuildSpecificAsset('metafoxLogoHorizontalDark')
               : './images/logo/metamask-logo-horizontal.svg'
           }
-          className={classnames(
-            'app-header__metafox-logo',
-            'app-header__metafox-logo--horizontal',
-          )}
+          className={classnames({
+            'app-header__metafox-logo': !isOnboarding,
+            'app-header__metafox-logo--horizontal': !isOnboarding,
+            'onboarding-app-header__metafox-logo': isOnboarding,
+            'onboarding-app-header__metafox-logo--horizontal': isOnboarding,
```

Comment:
Since the classes are used for only this component, is it possible to merge the css and use only 2 classes here ?

Reply:
Removed extraneous classnames. Good call @jpuri!